rocketpy/mathutils/function.py

```diff
@@ -3261,12 +3353,19 @@ def __validate_extrapolation(self, extrapolation):
 
         ## multiple dimensions
         elif self.__dom_dim__ > 1:
-            if extrapolation not in [None, "natural"]:
+            if extrapolation is None:
+                extrapolation = "natural"
+            if extrapolation == "natural" and self.__interpolation__ == "linear":
+                warnings.warn(
+                    "Extrapolation 'natural' is not supported for multidimensional "
+                    "linear interpolation. 'rbf' will be used to extrapolate."
```

Comment:
it says that rbf will be used, but I don't see where you are setting extrapolation to rbf

Reply:
No set is needed, since there is not "rbf" extrapolation option, only "constant", "natural" and "zero". The warning is there to point out that the "natural" extrapolation of the "linear" interpolation is an rbf algorithm, as I pointed out why is the PR description.

I am open to ideas on improving this, but I found the way it is implemented the most consistent regarding the current behavior of interpolation and extrapolation settings of 1-D Functions. Remember, there is no "spline" or "linear" extrapolation option for 1-D Functions, those are only supported through "natural".